packages/components/molecules/f-searchbox/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@justeat/f-searchbox",
   "description": "Fozzie Searchbox – Just Eat Takeaway Global Searchbox",
+  "tag": "beta",
```

Comment:
By adding this property to beta packages, we ensure that the package will be tagged correctly on the registry, and won't override the `latest` release